core/storage/voiceover/gae_models.py

```diff
@@ -45,6 +45,17 @@ class VoiceoverDict(TypedDict):
     duration_secs: float
 
 
+VoiceoversAndContentsMappingType = Dict[
```

Comment:
I don't think you need "Type" at the end? I mean we don't say DictType, ListType etc.

Or is having Type at the end a standard convention? Just wanted to double-check.